Resources/Maps/lighthouse.yml

```diff
@@ -45550,7 +45550,7 @@ entities:
   - canCollide: False
     type: Physics
 - uid: 3411
-  type: RightHandDrone
+  type: RightArmBorg

```

Comment:
is this cleared with ODJ

Reply:
I'll bring it up. It's only mapped on lighthouse